packages/kolibri-components/src/styles/colorsDefault.js

```diff
@@ -14,8 +14,8 @@ export const defaultTokenMapping = {
   fineLine: '#dedede',
   appBar: 'brand.primary.v_400',
   appBarDark: 'brand.primary.v_700',
-  appBarFullscreen: 'palette.grey.v_900',
-  appBarFullscreenDark: 'black',
+  appBarFullscreen: 'brand.primary.v_400',
+  appBarFullscreenDark: 'brand.primary.v_700',
```

Comment:
Note that there are two pairs of color tokens, `appBar` (which uses `brand.primary`) and `appBarFullscreen` (which uses black). In order to change the color of a particular app bar, we would want to point it the other token, not change the definitions here in `colorsDefault.js`

However this is moot given the mistake in the original issue I mentioned above.

Reply:
Ah makes sense, I'll update the PR. Any existing modal with that behavior? I want to make sure I'm fixing it the right way

Reply:
@indirectlylit done, I think I understand the problem now, reverted back to the previous colors and updated the immersivePage behavior for those modal pages.